### ISSUE TITLE
gnuplot: missing help file `gnuplot.gih`

### ISSUE THREAD

**Issue by hyperpallium** [termux/termux-packages#2764]
```
gnuplot> help plot
/data/data/com.termux/files/usr/share/gnuplot/5.2/gnuplot.gih: No such file or directory
```

The tricky thing is you can't just copy this `gih` file, it's generated with a program, `gnuplot-5.2.4/docs/doc2gih.c`. I'll see if I can work out how to do this, and make a PR.

**Comment by fornwall** [comment]
I think this change should do it: c8cdef550833203e66b21e0e3670e01bfe8a29c5

Can you update to the latest `5.2.4-1` version of the package by running `pkg up` and verify that it works as expected?

**Comment by fornwall** [comment]
Is there anything more from the `docs/` that should be included in the package?

**Comment by fornwall** [comment]
Thanks a lot!

The problem was not `make -C docs/ gnuplot.gih`, which works, but that the `termux_step_host_build()` function is not called on subsequent builds, as the host build is cached between builds, so the file was only installed on the first build

8ab448abb04924895df464209fd78fc0a880d4fc fixes that, and in the updated `5.2.4-2` version of the package the help command should now work for real!

**Comment by hyperpallium** [comment]
Great, thanks for following up so quickly, and it's working now!
